streams-tests/jvm/src/test/scala/zio/stream/ZStreamPlatformSpecificSpec.scala

```diff
@@ -174,22 +175,16 @@ object ZStreamPlatformSpecificSpec extends ZIOBaseSpec {
             assert(exit.untraced)(failsCause(containsCause(Cause.interrupt(selfId))))
         }
       ),
-      testM("fromQueue")(checkM(Gen.small(Gen.chunkOfN(_)(Gen.anyInt))) { c =>
-        for {
-          queue <- Queue.unbounded[Int]
-          _     <- queue.offerAll(c.toSeq)
-          fiber <- ZStream
-                    .fromQueue(queue)
-                    .foldWhileM[Any, Nothing, Int, List[Int]](List[Int]())(_ => true)((acc, el) =>
-                      IO.succeedNow(el :: acc)
-                    )
-                    .map(_.reverse)
-                    .fork
-          _     <- waitForSize(queue, -1)
-          _     <- queue.shutdown
-          items <- fiber.join
-        } yield assert(items)(equalTo(c.toSeq.toList))
-      })
+      testM("fromFile") {
+        val data = List.fill(76)("a").mkString
```

Comment:
We are are pulling the same data every time. Since we are dealing with mutable buffers, it makes the test less powerful.
Maybe `data = (1 to 100).mkString` ?

Reply:
Good point!